common/views.py

```diff
@@ -3,7 +3,7 @@
 from permissions import isPhoenix
 
 class GenericCRUDView(generics.GenericAPIView):
-    requireApiKey = True
+    requireApiKey = False
```

Comment:
This is unacceptable. You are affecting the execution mechanism for all its children